tools/dotnet-msidentity/Program.cs

```diff
@@ -246,39 +288,55 @@ private static async Task<int> HandleUpdateProject(ProvisioningToolOptions provi
 
         private static Option ClientSecretOption()=>
             new Option<string>(
-                aliases: new [] {"-cs", "--client-secret"},
+                aliases: new [] {"--client-secret"},
                 description: "Client secret to use as a client credential.")
             {
                 IsRequired = false
             };
 
+        private static Option RedirectUriOption() =>
+            new Option<IList<string>>(
+                aliases: new[] { "--redirect-uris" },
+                description: "Redirect URIs for web platform.")
```

Comment:
I'd suggest: "When specified, adds the provided redirect URIs as web redirect URIs for the app?"